src/link-utils.js

```diff
@@ -0,0 +1,35 @@
+const parseRawLinks = (rawLinks) => {
```

Comment:
For function declarations, I'd recommend utilizing the normal function declaration syntax unless you want to use this as a variable somewhere in this file.

Arrow functions are definitely recommended for things like anonymous functions though.

Reply:
Done

Reply:
Sure, if Jason wants it that way, that's fine. I haven't seen arrow functions used for declarations in many projects like this and do note that the two ways have different implications. However, for these particular functions, I don't think it will really matter.